src/coreclr/System.Private.CoreLib/src/System/Reflection/Metadata/AssemblyExtensions.cs

```diff
@@ -12,13 +12,21 @@ public static class AssemblyExtensions
         [return: MarshalAs(UnmanagedType.Bool)]
         private static extern unsafe bool InternalTryGetRawMetadata(QCallAssembly assembly, ref byte* blob, ref int length);
 
-        // Retrieves the metadata section of the assembly, for use with System.Reflection.Metadata.MetadataReader.
-        //   - Returns false upon failure. Metadata might not be available for some assemblies, such as AssemblyBuilder, .NET
-        //     native images, etc.
-        //   - Callers should not write to the metadata blob
-        //   - The metadata blob pointer will remain valid as long as the AssemblyLoadContext with which the assembly is
-        //     associated, is alive. The caller is responsible for keeping the assembly object alive while accessing the
-        //     metadata blob.
+        /// <summary>
+        /// Retrieves the metadata section of the assembly, for use with <see cref="T:System.Reflection.Metadata.MetadataReader" />.
+        /// </summary>
+        /// <param name="assembly">The assembly for which retrieve metadata.</param>
```

Comment:
I'm not a native english speaker but I think this is more correct:
```suggestion
        /// <param name="assembly">The assembly from which to retrieve the metadata.</param>
```